backend/app/modules/user/antisybil/service/initial.py

```diff
@@ -76,6 +85,26 @@ def _verify_address_is_not_delegated(self, user_address: str):
             raise AddressAlreadyDelegated()
 
 
+guest_list_stamp_providers = [
+    "AllowList#OctantFinal",
+    "AllowList#OctantEpochTwo",
```

Comment:
i don't know excatly how we should interpret the naming for stamps but isn't ...EpochTwo, ...EpochOne stamps only about precious epochs, not E4?

Reply:
+1